metrics/network/network-metrics-iperf3.sh

```diff
@@ -0,0 +1,540 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test measures the following network essentials:
+# - bandwith simplex
+# - bandwith duplex
+# - jitter
+#
+# These metrics/results will be got from the interconnection between
+# a client and a server using iperf tool.
```

Comment:
s/iperf/iperf3/ - as there are more than one variant of iperf out there iirc, so let's clarify.